src/GraphQLBackend.ts

```diff
@@ -141,5 +136,5 @@ export interface IGraphQLBackend {
   // Human readable schema that should be replaced with current one
   schema?: string,
   // Resolvers that should be mounted to schema`
-  resolvers?: object
+  resolvers?: string
```

Comment:
It is good to have this here as string for the moment but we might need to introduce another abstraction (outputformater)